documentation/pnpm scripts.md

```
@@ -0,0 +1,132 @@
+
+## pnpm generate_graphql_sdl_file
+
+This command is used to generate the graphql schema definition language file for talawa api's graphql implementation at the root directory of this workspace in a file named `schema.graphql`.
+
+## pnpm generate_gql_tada
+
+This command is used to generate the files containing the typescript types corresponding to talawa api's graphql implementation and ahead of time graphql document type cache corresponding to the graphql operation documents used in integration tests for talawa api's graphql implementation. This is done by inferring them from the graphql sdl file present at the root directory of this workspace in the file name `schema.graphql` and the graphql operation documents present within the the directory named `test` at the root directory of this workspace.
+
+More information can be found at these links:
+
+1. https://gql-tada.0no.co/reference/config-format#tadaoutputlocation
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)